bundles/org.openhab.binding.openthermgateway/src/main/java/org/openhab/binding/openthermgateway/internal/DataItem.java

```diff
@@ -0,0 +1,74 @@
+/**
+ * Copyright (c) 2010-2020 Contributors to the openHAB project
+ *
+ * See the NOTICE file(s) distributed with this work for additional
+ * information.
+ *
+ * This program and the accompanying materials are made available under the
+ * terms of the Eclipse Public License 2.0 which is available at
+ * http://www.eclipse.org/legal/epl-2.0
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ */
+package org.openhab.binding.openthermgateway.internal;
+
+import javax.measure.Unit;
+
+import org.eclipse.jdt.annotation.NonNullByDefault;
+import org.eclipse.jdt.annotation.Nullable;
+
+/**
+ * @author Arjen Korevaar - Initial contribution
+ */
+@NonNullByDefault
+public class DataItem {
+    private int id;
```

Comment:
It seems that a lot of these fields can be marked `final` I would always prefer to make class level fields explicitly final when possible.

Reply:
Prefer, or needed ?

Reply:
It will improve the code and ease future maintenance, it will thus benefit all of us. You will not find any maintainer who will object making fields `final`.

Reply:
That's not really the question though.. it's about what is needed to get this code up to a level of standard set to approve the PR. I don't mind spending some of my valuable time to share this binding with others.. but with every reviewer, new considerations are brought in which are fine by itself but make this PR process a never ending story.

Again, if changes are required, based on set standards, I will be happy to put more time in.